js/app.js

```diff
@@ -20,95 +20,194 @@ const logo = document.querySelector('.logo');
         description:"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
-        featuredImage:"../assets/images/Img-Plaholder.svg",
+        featuredImage:"assets/images/Img-Plaholder.svg",
         teachnologies:['Bootstrap', 'RSpec','Selenium', 'Ruby on Rails'],
         versionURL:"https://github.com/Bornittah/portifolio",
         sourceURL:"https://github.com/Bornittah/portifolio",
 
     },
     {
-        name:"Keeping track of hundreds of components",
+        title:"Keeping track of hundreds of components",
         description:"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
-        featuredImage:"../assets/images/Img-Plaholder.svg",
+        featuredImage:"assets/images/Img-Plaholder.svg",
         teachnologies:['Bootstrap', 'RSpec','Selenium', 'Ruby on Rails'],
         versionURL:"https://github.com/Bornittah/portifolio",
         sourceURL:"https://github.com/Bornittah/portifolio",
 
     },
     {
-        name:"Keeping track of hundreds of components",
+        title:"Keeping ",
         description:"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
-        featuredImage:"../assets/images/Img-Plaholder.svg",
+        featuredImage:"assets/images/Img-Plaholder.svg",
         teachnologies:['Bootstrap', 'RSpec','Selenium', 'Ruby on Rails'],
         versionURL:"https://github.com/Bornittah/portifolio",
         sourceURL:"https://github.com/Bornittah/portifolio",
 
     }
 ];
 
-let div=document.querySelector('.grid-container')
+let project='';
 for(let obj of projects){
     let tech=obj.teachnologies;
-    div+=`<div class="grid-item">
+    project+=`<div class="grid-item project">
     <div class="item-image">
         <img src="${obj.featuredImage}" alt="featured-image">
```

Comment:
- [ ] As the previous reviewer said your image is still not loaded. Kindly fix it.
- [ ] [Optional] kindly make `X` buttons same as figma design.
### your design
![his](https://user-images.githubusercontent.com/81727988/169632612-269496e8-e3e2-4b68-ac38-9b06260b2859.png)
### Figma design
![figg](https://user-images.githubusercontent.com/81727988/169632621-734bdd8b-daa4-457c-9c5e-297c0b302d5c.png)